app/subscriber/src/store/slices/content/interfaces/IContentState.ts

```diff
@@ -2,10 +2,12 @@ import {
   IContentListAdvancedFilter,
   IContentListFilter,
 } from 'features/content/list-view/interfaces';
-import { IContentModel, IPaged } from 'tno-core';
+import { IContentModel, IOptionItem, IPaged } from 'tno-core';
 
 export interface IContentState {
   filter: IContentListFilter;
   filterAdvanced: IContentListAdvancedFilter;
   content?: IPaged<IContentModel>;
+  dateFilter?: IOptionItem | null;
```

Comment:
Change to a single object for the `pressGalleryFilter` so that it's more intuitive when maintaining the store.